packages/tools/release-utils-internal/src/package-info.ts

```diff
@@ -8,7 +8,13 @@ import { Version, VersionString } from "./version";
 export interface PackageJsonFileContent {
     name?: string;
     widgetName?: string;
+    // User friendly name. This will be included in release title.
     moduleName?: string;
+    // Module name in Studio Pro (visible in App Explorer)
+    moduleNameInModeler?: string;
+    // Module folder name under themesources
+    moduleFolderNameInModeler?: string;
+
```

Comment:
I don't see that we have those fields in our `package.json` files, do we plan to introduce those fields?

Reply:
Yes, as I see this fields will make it easier to navigate through testProject and locate files that belongs to module